pkg/utils/events.go

```diff
@@ -0,0 +1,57 @@
+package utils
```

Comment:
It is more suitable to move this to `schema` package instead. Please think about how the users will use the library in other components.

I think it might be a good idea to expose the event names as constants in this package, what do you think? We can write a bash script to take event names from schema file and export it as constants in event_names.go file. 
So it will be more user friendly for the end user. For example, this is how it could be.

event_names.go (autogenerated)
```go
var (
  EventNetPolViolation = "NetPolViolation"
)
```

when imported in other components,

```go
import (
  ...monitoring/pkg/schema
)

...
e := event.NewEvent(schema.EventNetPolViolation, ...other args)
// record event
...

```